sqlagg/columns.py

```diff
@@ -89,6 +89,8 @@ class ArrayAggColumn(BaseColumn):
     """
     Perform array aggregation on a column
     Pass order_by_col to sort by another column within the group
+    Using order_by_col clause is not fully supported for partitioned databases in citus 
+    and should be used cautiously
     Example: array generated for a column col1, ordered by col2 with select clause like
     Select ARRAY_AGG(col1 ORDER BY col2), col3 ..
```

Comment:
nit: formatting change
```suggestion
    Perform array aggregation on a column.
    Pass order_by_col to sort by another column within the group.
    Example: array generated for a column col1, ordered by col2 with select clause like
    Select ARRAY_AGG(col1 ORDER BY col2), col3 ..
    
    Note: Using order_by_col clause is not fully supported for partitioned tables in CitusDB 
    and should be used cautiously.
```

Reply:
cool, thanks Simon.

In order to decide where to place the warning, I used the earlier order so that its along with the concerning column and should not get missed. Does that sound reasonable or any kind of Note/Warnings should be in the end as a practice?

Reply:
For specific cases like this I think it makes sense to keep it as a separate part of the docs. If it is a general warning and is broadly applicable then it could be part of the main docs. When I read it I just felt that it broke the flow of the docs.

Reply:
makes sense. thanks.